app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -562,10 +569,15 @@ public void onActivityResult(ActivityResult result) {
             skipOnce = savedInstanceState.getInt("skipOnce") == 1 ? true : false;
         }
         mCropperOptions = new UCrop.Options();
-        mCropperOptions.setCompressionFormat(Bitmap.CompressFormat.JPEG);
+        setCropperOptions();
         setCropperTheme();
     }
 
+    private void setCropperOptions(){
+        mCropperOptions.setCompressionFormat(Bitmap.CompressFormat.JPEG);
+        mCropperOptions.setFreeStyleCropEnabled(false);
```

Comment:
Personally I find free style crop make it a little janky because the crop area would prevent user from two finger pinching/rotating the image

Do toggle it around and test it however

Reply:
@TheLastProject @waffshappen 
https://github.com/Yalantis/uCrop/blob/281c8e6438d81f464d836fc6b500517144af264a/ucrop/src/main/java/com/yalantis/ucrop/UCrop.java#L238
play around it a bit if you have time,

Reply:
I personally very strongly prefer FreeStyleCrop being enabled. It is otherwise nearly impossible to crop the images as I want to.

Reply:
What about the bottom bar?
Meanwhile the toolbar "Edit Photo" text can also be changed
![image](https://user-images.githubusercontent.com/22017945/139771178-0bffa4e2-03af-4db0-8fcc-975d6379c68c.png)

Reply:
I like the bottom bar and I think the aspect ratio option you offer is actually great because the vast majority of loyalty cards are sized to fit in a standard wallet. I think offering 1:1, Custom and Card in that order (with Custom just being original, but hopefully making more obvious manual resizing is supported) makes most sense, but suggestions welcome.

Using "Set front image" for the title when editing the front image and things like that are indeed perfect.

This is looking really awesome and I really love you putting so much thought in all the little things to make it as perfect as it can be!

Reply:
Give the current state a try then
It offers the two aspect ratios and the one from the source image, at the same time free scale is enabled so one can change the crop window on the corners

You can also start reviewing the code for merge